kickstart/create-game.yaml

```
@@ -0,0 +1,30 @@
+summary: Create the $SRYYYY game
+
+priority: should
```

The game feels pretty important
```suggestion
priority: must
```